dotCMS/src/main/java/com/dotcms/contenttype/business/ContentTypeAPI.java

```diff
@@ -1,5 +1,6 @@
 package com.dotcms.contenttype.business;
 
+import com.dotcms.business.WrapInTransaction;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.business.WrapInTransaction'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)